src/DependencyInjection/UnleashEnvVarProcessor.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Unleash\Client\Bundle\DependencyInjection;
+
+use Symfony\Component\DependencyInjection\EnvVarProcessorInterface;
+use Unleash\Client\Unleash;
+
+final class UnleashEnvVarProcessor implements EnvVarProcessorInterface
+{
+    private Unleash $client;
+
+    public function __construct(Unleash $client)
+    {
+        $this->client = $client;
+    }
```

Comment:
Please use constructor promotion to be consistent with the rest of the code:

```suggestion
    public function __construct(private Unleash $client)
    {
    }
```